summit/2018.html

```diff
@@ -33,8 +33,8 @@ <h2 class="location">Tirana, Albania
         <img class="marker" src="/images/summit/map-marker.svg" width="55px" height="74px"
             alt="Location"/>
         <h2 class="location">Location</h2>
-        <p class="data">Venue still to be confirmed<br />Tirana, Albania</p>
-        <!--<img class="map" src="/images/summit/map.jpg" width="540px" height="313px" alt="Map"/>  -->
+        <p class="data"><strong>OFIÇINA</strong><br />Rruga Asim Vokshi<br />Tirana, Albania</p>
+        <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank" rel="noopener"><img class="map" src="/images/summit/map.jpg" width="600px" height="347px" alt="Map"/></a>
```

Comment:
Suggest making: <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank"

into 

        <a href="https://osm.org/go/xexSY_f5h--?m=" target="_new"

...

Reply:
Good thinking!

Reply:
@ei8fdb @elioqoshi thought I missed something – but "_new" does not exist ;) https://stackoverflow.com/a/4964223